src/components/MobileFilters.tsx

```diff
@@ -1,25 +1,29 @@
-import React, { useState } from 'react';
-import styled from '../styling/styled';
+import React, { useState, FC } from 'react';
 import PerfectScrollbar from 'react-perfect-scrollbar';
-import { Choice, FilterData } from '../data/filters';
 import { BsCheck, BsChevronLeft, BsChevronRight } from 'react-icons/bs';
 import { FiFilter } from 'react-icons/fi';
+import { Choice, FilterData } from '../data/filters';
+import styled from '../styling/styled';
 
-const FiltersButton = styled.button`
+const StyledButton = styled.button`
   border: none;
   background: transparent;
   display: flex;
   align-items: center;
+  outline: none;
   font-size: 1.2em;
+`;
 
+const FiltersButton = styled(StyledButton)`
   svg {
     color: ${(props) => props.theme.colors.primary};
     margin-right: 10px;
   }
 `;
+
 const MobileFiltersModal = styled.div<{ active: boolean }>`
   position: fixed;
-  top: 0;
+  top: 100px;
```

Comment:
^

Reply:
The filters menu is not visible in mobile version - you want to fix this in another issue?